src/store/ui/reducer.js

```diff
@@ -55,6 +56,21 @@ const uiReducer = (state = defaultState, action) => {
         openedNode: p.id === state.openedNode ? false : p.id
       }
     }
+    case 'UI_AUX_TOGGLE_OPEN': {
```

Comment:
I created a generic action/reducer thingy that allows things to be opened and closed globally. All it does is add or remove keys from an array. If a key is present in the array, we know this thing is "open". Currently using this for the "advanced" menu reveal for params. Could be used anywhere else, as long as the keys are always unique.